include/BasicTypes.hpp

```diff
@@ -101,7 +101,7 @@ enum NiFileVersion : uint32_t {
 	V20_6_5_0 = 0x14060500,
 	V30_0_0_2 = 0x1E000002,
 	V30_1_0_3 = 0x1E010003,
-	UNKNOWN = 0x7FFFFFFF
+	UNKNOWN = 0x7FFFFFFF // should probably be changed back to 0xFFFFFFFF
```

Comment:
I've left this in but added a macro to clarify. The SWIG code does not compile with the official value.

Reply:
How come? have you tried adding an unsigned literal suffix, ie ```0xFFFFFFFFU``` or a static cast? Or does it not recognize the underlying type?